src/domain/networking/packets/EntityData.ts

```diff
@@ -9,29 +9,1932 @@
+                            equippableRightRotationOffset,
+                            equippableIndicatorUrl,
+                            equippableIndicatorScale,
+                            equippableIndicatorOffset,
+                            density,
+                            velocity,
+                            angularVelocity,
+                            gravity,
+                            acceleration,
+                            damping,
+                            angularDampling,
+                            restitution,
+                            friction,
+                            lifetime,
+                            collisionless,
+                            collisionMask,
+                            dynamic,
+                            collisionSoundUrl,
+                            actionData,
+                            cloneable,
+                            cloneLifetime,
+                            cloneLimit,
+                            cloneDynamic,
+                            cloneAvatarIdentity,
+                            cloneOriginId,
+                            script,
+                            scriptTimestamp,
+                            serverScripts,
+                            itemName,
+                            itemDescription,
+                            itemCategories,
+                            itemArtist,
+                            itemLicense,
+                            limitedRun,
+                            marketplaceID,
+                            editionNumber,
+                            entityInstanceNumber,
+                            certificateID,
+                            certificateType,
+                            staticCertificateVersion,
+                            shapeType,
+                            compoundShapeUrl,
+                            color,
+                            textures,
+                            modelUrl,
+                            modelScale,
+                            jointRotationsSet,
+                            jointRotations,
+                            jointTranslationsSet,
+                            jointTranslations,
+                            groupCulled,
+                            relayParentJoints,
+                            blendShapeCoefficients,
+                            useOriginalPivot,
+                            animationUrl,
+                            animationAllowTranslation,
+                            animationFPS,
+                            animationFrameIndex,
+                            animationPlaying,
+                            animationLoop,
+                            animationFirstFrame,
+                            animationLastFrame,
+                            animationHold
+                        });
+                }
+                dataPosition += sectionLength;
+            }
+            dataPosition += entityMessagePosition;
```

Comment:
Having both of these updates to dataPosition isn't right?

Also, should assert that `sectionLength` matches up with entityMessagePosition, to help catch errors when implementing other entity types.